src/app/bug/shared/bug-data.service.spec.ts

```diff
@@ -22,4 +25,13 @@ describe('BugDataService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  describe('getBugs()', () => {
+    
+    it('should get the bug data from the data serviceAPI', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8845963)
```suggestion
    it("should get the bug data from the data serviceAPI", () => {
```